buổi-9/bai1/bai-1.js

```diff
@@ -0,0 +1,26 @@
+function tinhLuong(){
```

Comment:
```suggestion
function getSalary() {
```
nhìn xem dấu cách viết ntn
Không dùng tiếng việt

tất cả các hàm đều phải nhận vào params để tính toán, chứ không phải nhập từ `prompt`